PlayCover/AppInstaller/Installer.swift

```diff
@@ -35,6 +35,19 @@ class Installer {
         return response == .alertFirstButtonReturn
     }
 
+    static private func returnErrorString(error: Error) -> String {
+        switch error.localizedDescription {
+        case let str where str.contains("(disk full?)"):
+            return NSLocalizedString("alert.notSpace", comment: "")
```

Comment:
The switch statement doesn't need a `return` in it as by default it will return the value [SE-0380](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0380-if-switch-expressions.md).